darts/models/forecasting/linear_regression_model.py

```diff
@@ -129,7 +129,8 @@ def encode_year(idx):
             Default: ``None``.
         multi_models
             If True, a separate model will be trained for each future lag to predict. If False, a single model is
-            trained to predict at step 'output_chunk_length' in the future. Default: True.
+            trained to predict each step 'output_chunk_length' in the future (features lags are shifted for each step).
```

Comment:
Could we clarify a bit more which steps and by how much they are shifted back? Basically something lke that they are shifted back `output_chunk_length - n` steps for each step `n` in the output chunk.